docs/guides/first-shopify-abtest.mdx

```diff
@@ -0,0 +1,140 @@
+---
+sidebar_label: Your First Shopify A/B Test
+title: Build your first Shopify A/B test  
+---
+
+In this guide we'll run an A/B test to optimize the landing page hero banner of our Shopify store. Our hypothesis is that optimizing the hero banner will engage new visitors and returning buyers in a way that leads them to buy more from the store. We'll measure this using metrics that include 'add to cart' and purchase events from buyers. Let's dive in!
```

Comment:
```suggestion
In this guide we'll run an A/B test to optimize the landing page hero banner of our Shopify store. Our hypothesis is that optimizing the hero banner will engage new visitors and returning buyers in a way that leads them to buy more from the store. We'll measure this using metrics that include 'add to cart' and 'purchase event' from buyers. Let's dive in!
```

I don't see a reference to add to cart, but there's a reference to purchase event below.